public/JobHandler/callbacks/force_merge.tsx

```diff
@@ -0,0 +1,169 @@
+                          </li>
+                        ))}
+                        <li>Some shards are unassigned.</li>
+                        <li>
+                          Insufficient disk space: Force merging requires disk space to create a new, larger segment. If the disk does not
+                          have enough space, the merge process may fail.
+                        </li>
+                        <li>
+                          Index read-only: If the index is marked as read-only, a force merge operation cannot modify the index, and the
+                          merge process will fail.
+                        </li>
+                        <li>
+                          Too many open files: The operating system may limit the number of files that a process can have open
+                          simultaneously, and a force merge operation may exceed this limit, causing the merge process to fail.
+                        </li>
+                        <li>
+                          Index corruption: If the index is corrupted or has some inconsistencies, the force merge operation may fail.
+                        </li>
+                      </ul>
+                    </>
+                  }
+                />
+              ) as unknown) as string,
+            },
+            {
+              toastLifeTimeMs: 1000 * 60 * 60 * 24 * 5,
+            }
+          );
+        }
+      } else {
+        core.notifications.toasts.addDanger(
+          {
+            iconType: "alert",
+            title: ((
+              <>
+                Force merge from <FormatResourcesWithClusterInfo resources={extras.sourceIndex} clusterInfo={extras.clusterInfo} /> has some
+                errors, please check the errors below:
+              </>
+            ) as unknown) as string,
+            text: ((
+              <div style={{ maxHeight: "30vh", overflowY: "auto" }}>
+                <ul key="error.reason">
+                  <li>{error.reason}</li>
+                </ul>
+              </div>
+            ) as unknown) as string,
+          },
+          {
+            toastLifeTimeMs: 1000 * 60 * 60 * 24 * 5,
+          }
+        );
+      }
+      return true;
+    }
+  }
+
+  return false;
+};
+
+export const callbackForForceMergeTimeout: CallbackType = (job: ForceMergeJobMetaData, { core }) => {
+  const extras = job.extras;
+  if (extras.toastId) {
+    core.notifications.toasts.remove(extras.toastId);
+  }
+  core.notifications.toasts.addWarning(
+    {
+      title: ((
+        <>
+          Force merging <FormatResourcesWithClusterInfo resources={extras.sourceIndex} clusterInfo={extras.clusterInfo} /> has failed.
+        </>
+      ) as unknown) as string,
+      text: ((
+        <>
+          The force merge operation has taken more than one hour to complete. To see the latest status, use `GET /.tasks/_doc/
+          {extras.taskId}`
```

Comment:
It is better to use `GET /_tasks/<task_id>`, query system index is not recommended.

Reply:
Good point, will adjust this.